client/ayon_deadline/plugins/publish/global/collect_jobinfo.py

```diff
@@ -97,7 +97,9 @@ def _add_deadline_families(self, instance):
 
         # 'publish.hou' has different submit job plugin
         # TODO find out if we need separate submit publish job plugin
-        if "publish.hou" in all_families:
+        if "publish.hou" in all_families or (
+            "workfile_publish_on_farm" in all_families
+        ):
```

Comment:
I think this is more readable.
```suggestion
        if (
            "publish.hou" in all_families 
            or "workfile_publish_on_farm" in all_families
        ):
```